src/styles/components/review.scss

```diff
@@ -0,0 +1,11 @@
+.reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
```

Comment:
BEM blocks couldn't have any margins. You have BEM-mix class "card__reviews" in your html so you can use it to specify margin